doc/releases/reef.rst

```diff
@@ -0,0 +1,219 @@
+  complements the existing ``rbd_aio_readv`` and ``rbd_aio_writev`` methods.
+* RBD: The ``rbd device unmap`` command has gained a ``--namespace`` option.
+  Support for namespaces was added to RBD in Nautilus 14.2.0. It has been
+  possible to map and unmap images in namespaces using the `image-spec` syntax
+  since 14.2.0, but an option like this (similar to options available in most
+  other commands) was missing.
+* RBD: All rbd-mirror daemon perf counters became labeled and as such are now
+  emitted only by the new ``counter dump`` and ``counter schema`` commands. As
+  part of this conversion, many also got renamed to better disambiguate
+  journal-based and snapshot-based mirroring.
+
+* RBD: list-watchers C++ API (`Image::list_watchers`) now clears the passed
+  `std::list` before potentially appending to it, aligning with the semantics
+  of the corresponding C API (``rbd_watchers_list``).
+* RBD: Trailing newline in passphrase files (the ``<passphrase-file>`` argument
+  in the ``rbd encryption format`` command and the
+  ``--encryption-passphrase-file`` option in other commands) is no longer
+  stripped.
+* RBD: Support for layered client-side encryption has been added. Cloned images
+  can now be encrypted. Each cloned image can be encrypted with its own
+  encryption format and passphrase, potentially different from that of the
+  parent image. The efficient copy-on-write semantics intrinsic to unformatted
+  (regular) cloned images have been retained.
+
+RGW
+~~~
+
+* The RGW policy parser now rejects unknown principals by default. If you are
+  mirroring policies between RGW and AWS, you might want to set
+  ``rgw_policy_reject_invalid_principals`` to ``false``. This change affects
+  only newly set policies, not policies that are already in place.
+* RGW's default back end for ``rgw_enable_ops_log`` has changed from ``RADOS``
+  to ``file``. The default value of ``rgw_ops_log_rados`` is now ``false``, and
+  ``rgw_ops_log_file_path`` now defaults to
+  ``/var/log/ceph/ops-log-$cluster-$name.log``.
+* RGW's pubsub interface now returns boolean fields using ``bool``. Before 
+  this change, ``/topics/<topic-name>`` returned ``stored_secret`` and
+  ``persistent`` using a string of ``true`` or ``false`` with quotes around
+  them. After this change, these fields are returned without quotes so they can
+  be decoded as boolean values in JSON. This also applies to the
+  ``is_truncated`` field returned by ``/subscriptions/<sub-name>``.
+* RGW's response of ``Action=GetTopicAttributes&TopicArn=<topic-arn>`` REST 
+  API now returns ``HasStoredSecret`` and ``Persistent`` as boolean in the JSON
+  string that is encoded in ``Attributes/EndPoint``.
+* All boolean fields that were previously rendered as strings by the 
+  ``rgw-admin`` command when the JSON format was used are now rendered as
+  boolean. If your scripts and tools rely on this behavior, update them
+  accordingly. The impacted field names are:
```

Comment:
@zdover23 - Edited/reviewed version from 2023 June 18: The following is a list of the field names impacted by this change: